src/wallet/mod.rs

```diff
@@ -6,7 +6,10 @@ use std::{
 };
 
 use bdk::{
-    bitcoin::{consensus::Encodable, hashes::Hash, BlockHash, Network},
+    bitcoin::{
+        consensus::Encodable, hashes::Hash, psbt::PartiallySignedTransaction, script::PushBytesBuf,
```

Comment:
Please use anonymous imports for traits with names like `Hash` that could conflict with other symbols. ie.
```
consensus::Encodable as _, hashes::Hash as _, ...
```